businesstime/australia.py

```diff
@@ -0,0 +1,101 @@
+
+from datetime import date
+from businesstime import Holidays
+
+class QueenslandPublicHolidays(Holidays):
+    """
+    http://www.qld.gov.au/recreation/travel/holidays/public/
+
+    Current coverage is only 2013-2016.
+    """
+
+    # Australian public holidays are legislated by each state individually for 
```

Comment:
Seems like this would mean we need to update this going forward (which would suck).

Could we instead have some sort of adapter pattern where you pass in coverage ranges and a set of holidays so that users can update this in their code?